Cannot find type definition file for 'marked'.

**Describe the bug**
Hi,
I'm getting the following error trying to compile a typescript project with easymde:
```
node_modules/easymde/types/easymde.d.ts(23,23): error TS2688: Cannot find type definition file for 'marked'.
```

**To Reproduce**
Steps to reproduce the behavior:
1. npm install easymde
2. Run tsc

**Expected behavior**
No TS errors

**Version information**
- OS: Debian testing
- Browser: N/A
- EasyMDE version: 1.24.0

--- a/package.json
+++ b/package.json
@@ -19,13 +19,13 @@
     "license": "MIT",
     "author": "Jeroen Akkerman",
     "dependencies": {
+        "@types/codemirror": "0.0.108",
+        "@types/marked": "^2.0.1",
         "codemirror": "^5.60.0",
         "codemirror-spell-checker": "1.1.2",
         "marked": "^2.0.1"
     },
     "devDependencies": {
-        "@types/codemirror": "0.0.108",
-        "@types/marked": "^2.0.1",
         "browserify": "^17.0.0",
         "gulp": "^4.0.2",
         "gulp-clean-css": "^4.2.0",

